libraries/SITL/SITL.cpp

```diff
@@ -499,7 +499,7 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
     // @Param: GPS_TYPE
     // @DisplayName: GPS 1 type
     // @Description: Sets the type of simulation used for GPS 1
-    // @Values: 0:None, 1:UBlox, 5:NMEA, 6:SBP, 7:File, 8:Nova, 9:SBP, 10:Trimble, 19:MSP
+    // @Values: 0:None, 1:UBlox, 5:NMEA, 6:SBP, 7:File, 8:Nova, 9:SBP2, 11:Trimble, 19:MSP
```

Comment:
isn't trimble a SBF type ? so we that would be 10 ? @Ryanf55

Reply:
No, Trimble PX1 uses GSOF as a protocol and has no affiliation with SBF.

Reply:
Ok so some comment on ap_gps are misleading then as they said sbf and trimble are similar